documentation/user_docs/docs/manual/Extracting_data_from_SQW_files_and_objects.rst

```diff
@@ -45,30 +75,82 @@ object stored in memory from which the pixels will be taken.
 Projection
 ----------
 
-This defines the coordinate and thus binning system you will use to plot the
+This defines the coordinate and thus binning system you will use to plot and analyse the
 data.
 
-``proj`` should be an instance of a ``projection`` (such as ``line_proj``,
-``sphere_proj``, etc.) containing information about the axes and the coordinate
-system you wish to use to plot the data.
+``proj`` should be an child of a ``aProjectionBase`` class (such as ``line_proj``,
+``sphere_proj``, etc.) containing information about the the coordinate
+system you wish to use to plot and analyse the data.
 
 .. note::
 
-   Because each point in the ``sqw`` file is labelled with h, k, and l (the
-   reciprocal lattice vectors) and energy, the underlying pixels will be
-   unchanged. It is possible to redefine the coordinate system with the one of
+   Because each point of ``sqw.pix`` data describes the
+   position in the reciprocal space and energy transfer, the underlying pixels will be
+   unchanged. For image (``dnd`` object) it is possible to redefine the coordinate system with the one of
    your choice; the projection merely describes how pixels will be accumulated
-   (binned) and thus displayed.
+   (binned) and thus displayed in image coordinate system.
```

Comment:
It would be good at this point to summarise the projection possibilities. Line proj is old but its packaging in this class is new. Spherical and cylindrical projections are new. The text below can then fill in the detail, but saying up front is useful to the reader so they know what the options are.

At line 78, the phrase "and thus binning system" may cause problems. It is not obvious that coordinates and binning should be rolled up together. An explanation of how binning info is layered onto the coordinate system would be helpful, particularly as you have gone to a lot of trouble to add detail on several other areas.

Reply:
_I would agree with your first suggestion but leave it for the next iteration. I feel need to provide short description of the projection (or rather `proj_info`) and go to the samples later. The actual contents will depend on how much we will progress with developting `proj_info`_
_To address second suggestion, I've modified the centence as follows:_

This defines the coordinate and thus the meaning of the binning you provided as arguments to `cut` and  will use to plot and analyse the data.

Reply:
Actually would tro to modify doc to address the first suggestion too.

Reply:
Please do also address the first comment. While large re-edits might be able to wait until April, it is important that the users reading this documentation are aware of what is now possible. Leaving it buried in the detail is not an option